shared/feature/feedings/src/main/java/com/epmedu/animeal/feedings/presentation/ui/FeedingItemSheetContent.kt

```diff
@@ -47,39 +50,52 @@ internal fun FeedingItemSheetContent(
     ) {
         FeedingItemSheetHeading(feeding)
         HeightSpacer(height = 24.dp)
-        Box(
+        Column(
             modifier = Modifier
-                .weight(1f)
-                .alpha(contentAlpha)
-                .align(Alignment.CenterHorizontally)
-                .padding(horizontal = 30.dp)
-                .clip(RoundedCornerShape(8.dp))
+                .fillMaxSize()
+                .verticalScroll(rememberScrollState())
         ) {
-            AsyncImage(
-                model = ImageRequest.Builder(LocalContext.current)
-                    .data(currentPhoto.url)
-                    .diskCacheKey(currentPhoto.name)
-                    .crossfade(true)
-                    .build(),
-                contentScale = ContentScale.Fit,
-                contentDescription = null,
-                onState = { state ->
-                    isPhotoLoading = state is Loading
+            Box(
+                modifier = Modifier
+                    .height(402.dp)
```

Comment:
Can we replace the fixed height here with some more adaptive solution to have visible carousel on smaller screens?

Reply:
Updated